src/hooks/useHandleExceedMaxCommentLength.ts

```diff
@@ -19,7 +19,7 @@ const useHandleExceedMaxCommentLength = () => {
         [hasExceededMaxCommentLength],
     );
 
-    const validateCommentMaxLength = useMemo(() => _.debounce(handleValueChange, 1500), [handleValueChange]);
+    const validateCommentMaxLength = useMemo(() => _.debounce(handleValueChange, 500, {leading: true}), [handleValueChange]);
```

Comment:
@aswin-s I don't think we need to reduce the timeout. Any reason to do it?

Reply:
1.5 seconds feels a bit too much in my testing. It waits for 1.5 seconds after last invocation to trigger the check. 500ms felt like a safe middle ground. The issue gets fixed even without the change in timing.

Reply:
@abdulrahuman5196 Do you think it is safe to revert the timeout change?

Reply:
I think we should only do the computation once the user stops typing and gives a pause, since it could add computational overhead to compute the error sooner. That's why I assume we kept it at 1500 ms. I think for this PR, it should be fine to leave it as it is.

Reply:
@aswin-s

Reply:
@abdulrahuman5196 I've reverted the timeout change.